src/pages/LoginPage/LoginPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+import { LoginLayout } from 'components/LoginLayout/LoginLayout'
+
+import styles from './LoginPage.module.scss'
+
+export const LoginPage = () => {
+  const [form, setValues] = useState({
+    username: '',
+    password: '',
+  })
+
+  const handleSubmit = e => {
+    e.preventDefault()
+
+    // console.log(form)
+  }
+
+  const updateField = e => {
```

Comment:
Use destructing for nested object keys. Example: ` ({ target: { name, value } }) => `